air-interpreter/src/wasm_bindgen.rs

```diff
@@ -41,8 +41,10 @@ pub const DEFAULT_LOG_LEVEL: LevelFilter = LevelFilter::Trace;
 pub fn main() {
     // it's necessary to initialize it with the minimal allowed log level,
     // because otherwise it's impossible to set less level than used during initialization.
+
```

Comment:
Looks like we forgot to remove this compilation target. Maybe we can just remove it in this PR, @mikevoronov what do you think?

Reply:
I think removing this file is too important to be merged with other PR.